pyaedt/modules/Material.py

```diff
@@ -779,6 +868,13 @@ def poissons_ratio(self, value):
 
     @property
     def diffusivity(self):
+        """Diffusivity.
+        
+        Returns
+        -------
+        type
+            Diffusivity of the material.
+        """
```

Comment:
@PipKat, I agree with how you did the docstrings here for getters/setters.  Thanks!